Controller/articles.contoller.js

```diff
@@ -31,4 +31,13 @@ exports.getArticleCommentsById = (req, res, next) => {
     selectArticleCommentsById(article_id).then(results => {
         res.status(200).send({comments: results})
     }).catch(next)
-}
\ No newline at end of file
+}
+
+exports.postArticleComment = (req, res, next) => {
+    const {username} = req.body
+    const {body} = req.body
+    const {article_id} = req.params
+    insertComment(username, body, article_id, next).then(results => {
```

Comment:
why have you passed next into the model?

Reply:
i thought i might need to do a .catch in the model but realised i didnt and forgot to remove it